internal/rbd/replicationcontrollerserver.go

```diff
@@ -47,7 +48,7 @@ type imageMirroringState string
 const (
 	// If the state is up+unknown means the rbd-mirror daemon is
 	// running and the image is demoted on both the clusters.
-	upAndUnknown imageMirroringState = "up+unknown"
+	unknown imageMirroringState = "unknown"
```

Comment:
the comment above still refers to "up+unknown"

Reply:
oops. Noted